blocks/library/post-title/post-title-edit.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import Editable from '../../editable';
+import PostPermaLink from './post-permalink';
+import { editPost } from '../../../editor/actions';
+
+const PostTitleEdit = ( { title, focus, setFocus, onChange } ) => [
+	focus && <PostPermaLink />,
+	<Editable
```

Comment:
I don't think this needs to be editable, just a plain `input` (we don't need to / cannot support rich text).